mlflow/sklearn/__init__.py

```diff
@@ -654,6 +655,11 @@ def fetch_logged_data(run_id):
     :param silent: If ``True``, suppress all event logs and warnings from MLflow during scikit-learn
                    autologging. If ``False``, show all events and warnings during scikit-learn
                    autologging.
+    :param max_hyper_param_runs: The max number of child mlflow runs to be created for
+                                 hyper parameter search based estimators. If one only cares about
```

Comment:
```suggestion
    :param max_hyper_param_runs: The maximum number of child MLflow runs to be created for
                                 hyperparameter search estimators. If one only cares about
```

Reply:
Can we also add documentation indicating that the user can specify `None` to omit a maximum limit and log a child run for each set of explored hyperparameters?